test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -48,7 +48,9 @@ OPT_HAS_ARG['d']=1         OPT_VAL['d']=3
 OPT_NAME['p']='pulseaudio'   OPT_DESC['p']='disable pulseaudio on the test process'
 OPT_HAS_ARG['p']=0             OPT_VAL['p']=1
 
-func_opt_parse_option "$@"
+fxunc_opt_parse_option "$@"
```

Comment:
While fixing this typo, can you also add `set -e` if testing is OK?

If `set -e` requires some other changes then forget it, does not really belong to this PR.